tests/app/forms/test_fields.py

```diff
@@ -1,22 +1,20 @@
-from wtforms import validators, StringField, FormField, SelectField
+from wtforms import StringField, FormField, SelectField
```

Comment:
This file seems to be testing field handlers more than fields, so should be renamed.